pyais/messages.py

```diff
@@ -590,7 +593,7 @@ class MessageType5(Payload):
     day = bit_field(5, int, default=0)
     hour = bit_field(5, int, default=0)
     minute = bit_field(6, int, default=0)
-    draught = bit_field(8, int, from_converter=from_course, to_converter=to_course, default=0)
+    draught = bit_field(8, float, from_converter=from_course, to_converter=to_course, default=0)
```

Comment:
should this have its own converter functions?

Reply:
Well, it would be a duplicate function that is identical to `to_course`. Instead I renamed `to_course` to `from_10th` and `to_10th`.